components/listingsPage/filters.js

```diff
@@ -240,9 +240,62 @@ const Filters = ({ t }) => {
     changeSingleShow(false);
     changeNote(true);
   };
+    // NONFRONTEND CODE STARTS
+
+  // setup alternate values for form input to trigger checked attr
+  let altAge = t("headings.age_at_time_of_travel");
+  let altTripType = "";
+  let altCoverArea = "";
+  let altCruise= "";
+  let altWhen = "";
+  let altCoverOngoingTreatment = "";
+  let altTerminalPrognnosis = "";
+  let altCoverSpecialEquipment = "";
+  let altAnnualOption = "";
+  let altSingleOption = "";
+
+  // Code to repopulate the input from the url
+ 
+  for (const property in query) {
+    if (`${property}` === "cruise_30_days_max_age") {
```

Comment:
I'm not sure if this is correct (though i'm not sure). Leastways I'm not seeing the age persist when the form is submitted - it doesn't appear in the URL when other filters are changed.

Reply:
When the form is submitted, it is actually deleted `(line 34 of server.js`) because it is not need in the following iterations. It is not until the page reload (after submission) that `Altage` is reassigned. So it wont show in the url. And this is because there is no age as an input in the database, the value is being assigned to `cruise_30_days_max_age`, `cruise_45_days_max_age`, `cruise_55_days_max_age`, l`and_30_days_max_age`, `land_45_days_max_age`, and ` land_55_days_max_age`.

Reply:
Ah I see, that make sense. 👍